### ISSUE TITLE
Chrome Process Fails To Hook When The `--auto-open-devtools-for-tabs` Flag Is Used

### ISSUE THREAD

**Issue by CanadianEngineer** [Microsoft/vscode-chrome-debug#470]
Hey everyone,

TL;DR: Adding `--auto-open-devtools-for-tabs` chrome argument breaks the chrome debugger by not allowing VS Code to hook the process. Launches as expected, but times out and closes.

I've recently started using the chrome debugger and while using VS Code to debug is pretty sweet, I've realized that some error messages don't show up in the VS code debugger console, and only show up in the Chrome debugger. I've looked online, and found that I can use the flag: `--auto-open-devtools-for-tabs`.

I add the flag to my `launch.json` file, and Chrome starts as expected with the dev tools open. However, CS Code fails to hook onto the Chrome process and I get a timeout error. I've also noticed that if I launch chrome through VS code normally, the chrome process remains attached until I open the debugging tools.

- VS Code Version:1.14.2

## `launch.json` file - Normal boot ##

```
{
	"version": "0.0.0",
	"configurations": [
		{
			"type": "chrome",
			"request": "launch",
			"name": "Launch Chrome",
			"url": "http://localhost:4200",
			"webRoot": "${workspaceRoot}",
			"sourceMaps": true,
			"userDataDir": "${workspaceRoot}/.vscode/chrome",
			"runtimeArgs": [
				"--disable-session-crashed-bubble",
				// "--auto-open-devtools-for-tabs",	// Doesn't attach properly when this argument is set
				"--remote-debugging-port=9222"
			]
		},
		{
			"name": "Attach Chrome",
			"type": "chrome",
			"request": "attach",
			"url": "http://localhost:4200",
			"port": 9222,
			"webRoot": "${workspaceRoot}",
			"sourceMaps": true
		}
	]
}
```
## `launch.json` attempting to open devtools, and failing to hook ##
```
{
	"version": "0.0.0",
	"configurations": [
		{
			"type": "chrome",
			"request": "launch",
			"name": "Launch Chrome",
			"url": "http://localhost:4200",
			"webRoot": "${workspaceRoot}",
			"sourceMaps": true,
			"userDataDir": "${workspaceRoot}/.vscode/chrome",
			"runtimeArgs": [
				"--disable-session-crashed-bubble",
				"--auto-open-devtools-for-tabs",
				"--remote-debugging-port=9222"
			]
		},
		{
			"name": "Attach Chrome",
			"type": "chrome",
			"request": "attach",
			"url": "http://localhost:4200",
			"port": 9222,
			"webRoot": "${workspaceRoot}",
			"sourceMaps": true
		}
	]
}
```

**Comment by roblourens** [comment]
That's right, only one thing can debug a tab at a time, so if the devtools are opened, vscode will be kicked off.

**Comment by auchenberg** [comment]
@CanadianEngineer What kind of errors are you missing from from the VS Code console? Would it be possible for you to open a new issue with the a log or something?
